### DISCUSSION TITLE
ChatInputArea - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#30] [Q&A]
# ChatInputArea - Lobe UI

The ChatInputArea component is used to display a chat input area with expandable and collapsible feature, and a send button to submit the chat message. It can be customized with actions and footer, and also supports input value change and composition event for Chinese input.

https://ui.lobehub.com/components/chat-input-area

**Comment by rdmclin2** [comment]
Need value control, thank you ：）

**Reply by canisminor1990** [reply]
Done 6c36a2516d93673fd7295d98d028986f5fc5932e

**Comment by TengFeiHao** [comment]
？？
